modules/developer_manual/examples/core/webdav_api/trashbin/list-files-in-trashbin-success-response.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0"?>
+<d:multistatus xmlns:d="DAV:" xmlns:s="http://sabredav.org/ns" xmlns:oc="http://owncloud.org/ns">
+  <d:response>
+    <d:href>/remote.php/dav/trash-bin/admin/</d:href>
```

Comment:
shouldn't that be ${USERNAME} instead of admin - to match the request?

Reply:
Right you are.